airbyte-integrations/connectors/source-amazon-seller-partner/source_amazon_seller_partner/schemas/GET_FBA_STORAGE_FEE_CHARGES_DATA.json

```diff
@@ -85,16 +85,16 @@
     "qualifies_for_inventory_discount": {
       "type": ["null", "string"]
     },
-    "storage-utilization-ratio": {
+    "storage_utilization_ratio": {
```

Comment:
Is it braking change?

Reply:
I would say yes if there were active users of the stream on cloud (based on this [page](https://docs.airbyte.com/contributing-to-airbyte/resources/pull-requests-handbook/#examples)). Since there are no customers, I decided not to mark this PR as a breaking change, however, based on @tolik0's comment above, this might affect OSS users. I don't know if the same logic of connector upgrade applies to them, so could you please advice if I should consider this as a breaking change?

Reply:
Yes, let's mark this as braking changes.

Reply:
Updated, please take a look